pages/docs/index.js

```diff
@@ -277,6 +277,18 @@ export default function DocsIndex() {
       title: "Extra",
       description: "More information",
       pages: [
+        
+        {
+          name: "Profile Tips",
+          path: "/docs/profile-tips",
+          description:
+          "Want some tips on how to make your Linkfree profile stand out? Click here! :)",
```

Comment:
Keep consistency with the existing style of writing:
```suggestion
          "Tips on making your LinkFree profile stand out.",
```